code/io/timer.cpp

```diff
@@ -104,7 +106,12 @@ std::uint64_t timer_get_nanoseconds()
 {
 	auto time = get_performance_counter();
```

Comment:
It's not necessarily in nanoseconds since the unit depends on what `SDL_GetPerformanceFrequency()` returns but on Linux this is in nanoseconds so there is an issue with possible overflows.